src/pcd.ts

```diff
@@ -78,8 +79,20 @@ function getVerifyKey() {
   return verifyKey
 }
 
-export async function verify(pcd: IdentityPCD): Promise<boolean> {
-  const vk = getVerifyKey()
+export async function verify(
+  pcd: IdentityPCD,
+  isWebEnv?: boolean
+): Promise<boolean> {
+  let vk
+  if (isWebEnv === true) {
```

Comment:
I don't think we should fetch verify key from another URL.

Reply:
How do you think we should handle that?

I was thinking that the web verifying and proving should rely on the same "state", and by so using the same artifacts stored on web.

Reply:
What do you mean by mock server?

For now I'm using aws S3

Reply:
I mean a location server to store proving key and we can use key have same "state" with artifacts folder.

Reply:
Yes that's a good idea!!
I added a ticket here => https://github.com/privacy-scaling-explorations/anon-aadhaar/issues/44

Can we validate this PR related to the web prover and add the mock server on the other ticket?

Reply:
I will approve PR after resolve conflict.